### DISCUSSION TITLE
DOCKER_HOST broken for Podman.sock

### DISCUSSION THREAD

**Discussion by constexpr-gsalazar** [microsoft/vscode-docker#4315] [Q&A] [answered]
Hello, I am currently using podman and I moved the docker command to be podman in the json.settings. Moreover, the DOCKER_HOST to point to the podman.sock. This works with version v1.22.2 however not on the latest version. Nor anything above 1.23 it seems. Is this feature broken ? Or am I missing something?

**Comment by bwateratmsft** [comment, marked as answer]
Starting in version 1.23.0, we began using exclusively the CLI for communication with Docker, which broke Podman. We are tracking that under #3241. For now, I'd recommend rolling back to 1.22.2.

**Reply by tony-sol** [reply]
https://github.com/microsoft/vscode-docker/issues/3241 is about almost 3.5 y.o. now, sad :c
